apple/validator.go

```diff
@@ -48,9 +54,10 @@ func New() *Client {
 }
 
 // NewWithURL creates a Client object with a custom URL provided
-func NewWithURL(url string) *Client {
+func NewWithURL(validationURL string, revokeURL string) *Client {
```

Comment:
Ugh I hate breaking changes but it makes sense to do that. Let me make a release before this PR gets merged so that we can have people lock into the older version if needed